src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/DebugPropertyPage/RemoveStartArgumentsActionHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Xml;
+using Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.Designer;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties;
+
+[Export(typeof(ILinkActionHandler))]
+[ExportMetadata("CommandName", "RemoveStartArguments")]
+internal class RemoveStartArgumentsActionHandler : ILinkActionHandler
+{
+    private const string PropertyNameToDelete = "StartArguments";
+    private const string UserSuffix = ".user";
+    private string? _xmlFile;
+
+    public Task HandleAsync(UnconfiguredProject project, IReadOnlyDictionary<string, string> editorMetadata)
+    {
+        _xmlFile = project.FullPath + UserSuffix;
```

Comment:
Please check with the CPS team about whether it would be better to use CPS APIs to modify the .user file, rather than doing direct XML-to-disk changes here.

Reply:
The cps apis marked this property as read only and doesn't allow to make changes to it.

Reply:
The .user file is part of the loaded project; in general we shouldn't modify directly.

Do we understand why "StartArguments" is being treated as read-only?